greymatter/modules/sidecar/tasks.tf

```diff
@@ -0,0 +1,73 @@
+
+# task definitions
+
+resource "aws_ecs_task_definition" "sidecar-task" {
+  family                   = "${var.name}-sidecar"
+  container_definitions    = local.sidecar_container
+  requires_compatibilities = ["EC2"]
+  network_mode             = "bridge"
+  cpu                      = "128"
+  memory                   = "128"
+  execution_role_arn       = var.execution_role_arn
+  task_role_arn            = var.execution_role_arn
+}
+
+# task defs with variables defined here:
+
+
+locals {
+  sidecar_container = <<DEFINITION
+[
+        {
+        "memoryReservation": 128,
+        "name": "${var.name}-sidecar",
+        "logConfiguration": {
+            "logDriver": "awslogs",
+            "options": {
+                "awslogs-group": "openjobs",
```

Comment:
I don't think there's terraform code that creates this openjobs cloudwatch log group.  Maybe it should be added somewhere in the infrastructure module?

Reply:
im not sure what you mean by this? I dont have any experience using cloudwatch really, but the log group openjobs does appear for me when i run this and the logs for control, control-api, control-api sidecar and edge do appear

Reply:
So I had to manually create a log group called "openjobs" in cloudwatch before this worked for me.  I think you maybe had openjobs created when you tried running my old ansible scripts (my best guess).  Anyway, if you go to `cloudwatch -> logs -> log groups` that's where it lives.  A quick resource like https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_log_group should create it.